### PR TITLE
Small fixes

### PR DESCRIPTION
**Pull request by OskarStark**
I am targetting this branch, because it is BC and does not change any code behaviour

## Subject
avoid long lines and use `NEXT_MAJOR` instead of `@todo`

### CODE REVIEW

**Comment by greg0ire** [comment]
LGTM

**Comment by OskarStark** [comment]
I updated the whole PR, please review again @greg0ire

**Comment by greg0ire** [comment]
Can you fix StyleCI ?

File: Command/Validators.php [hunk 1]
```diff
@@ -50,7 +50,10 @@ public static function validateEntityName($shortcut)
         $entity = str_replace('/', '\\', $shortcut);
 
         if (false === $pos = strpos($entity, ':')) {
-            throw new \InvalidArgumentException(sprintf('The entity name must contain a : ("%s" given, expecting something like AcmeBlogBundle:Post)', $entity));
+            throw new \InvalidArgumentException(sprintf(
+                'The entity name must contain a : ("%s" given, expecting something like AcmeBlogBundle:Post)',
```
Comment (greg0ire) [inline Command/Validators.php]: While you're at it : ":" => ": sign" or "colon sign", otherwise the error message is a bit hard to understand

Comment (greg0ire) [inline Command/Validators.php]: Same below

Comment (OskarStark) [inline Command/Validators.php]: i don't really get what you want, can you show me a diff for this line?

```diff
- foo
+ bar
```

Comment (greg0ire) [inline Command/Validators.php]:
```diff
The entity name must contain a : 
The entity name must contain a : sign
```

or

```diff
The entity name must contain a : 
The entity name must contain a colon sign
```

What you feel is best.

Comment (OskarStark) [inline Command/Validators.php]: done @greg0ire

File: Command/Validators.php [hunk 2]
```diff
@@ -51,7 +51,8 @@ public static function validateEntityName($shortcut)
 
         if (false === $pos = strpos($entity, ':')) {
             throw new \InvalidArgumentException(sprintf(
-                'The entity name must contain a : ("%s" given, expecting something like AcmeBlogBundle:Post)',
+                'The entity name must contain a ":" (cologn sign) '
```
Comment (greg0ire) [inline Command/Validators.php]: Das ist kein Köln sign ^^ it's splet "colon"

Comment (OskarStark) [inline Command/Validators.php]: haha 🎯

Comment (OskarStark) [inline Command/Validators.php]: fixed

**Comment by greg0ire** [comment]
> Can you fix StyleCI ?

Please?

**Comment by OskarStark** [comment]
@greg0ire do we relly add a blank line between next major and the phpdoc?

**Comment by greg0ire** [comment]
> @greg0ire do we relly add a blank line between next major and the phpdoc?

I usually merge both comments into one.

**Comment by OskarStark** [comment]
do you have an example for me?

**Comment by greg0ire** [comment]
```
    /**
     * NEXT_MAJOR : remove this method.
     *
     * @return BreadcrumbsBuilderInterface
     */
```

**Comment by OskarStark** [comment]
ok wil do this

**Comment by OskarStark** [comment]
wait, what is this?
https://github.com/sonata-project/SonataAdminBundle/blame/3.x/CONTRIBUTING.md#L185

😕

**Comment by greg0ire** [comment]
I think this should be changed :P

**Comment by OskarStark** [comment]
i created a PR in dev-kit

**Comment by OskarStark** [comment]
done @greg0ire 👍

**Comment by greg0ire** [comment]
Can you `git apply` the styleCI patch?

**Comment by OskarStark** [comment]
done

**Comment by greg0ire** [comment]
LGTSM

**Pull request merged**